### ISSUE TITLE
Python package is missing cloudclient

### ISSUE THREAD

**Issue by martenlehmann** [Weebly/cloud-client-python#2]
The whole subfolder https://github.com/Weebly/cloud-client-python/tree/master/weeblycloud/cloudclient is not packaged.

And as you're at it, please add the missing colon after `else`:

```>>> import weeblycloud
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vagrant/dev/weebly/weeblycloud/__init__.py", line 7, in <module>
    from .Account import *
  File "/home/vagrant/dev/weebly/weeblycloud/Account.py", line 1, in <module>
    from .SaveableResource import *
  File "/home/vagrant/dev/weebly/weeblycloud/SaveableResource.py", line 1, in <module>
    from .CloudResource import *
  File "/home/vagrant/dev/weebly/weeblycloud/CloudResource.py", line 31
    else
       ^
SyntaxError: invalid syntax
```

**Comment by slevi2103** [comment]
The bug is because missing semicolon on else.
https://github.com/Weebly/cloud-client-python/blob/master/weeblycloud/CloudResource.py#L31

**Comment by szainmehdi** [comment]
Closed by #3
